modules/plot/main/coffee/Expose.coffee

```diff
@@ -7,6 +7,14 @@ hx.plot.arcCurve = arcCurve
 hx.plot.svgCurve = svgCurve
 hx.Axis = Axis
 hx.Graph = Graph
+
+hx.graph = (options) ->
+  selection = new hx.detached 'div'
+  graph = new Graph selection.node(), options
+  # There is no point rendering it now, the selection is of zero size.
+  selection.on 'resize', 'hx.plot', -> graph.render()
```

Comment:
This line isn't needed. the hx.Graph constructor already does this.

The previous comment was more for future reference about namespacing events